cobalt/build/configs/initialize_variables.gni

```diff
@@ -34,9 +34,6 @@ declare_args() {
   build_with_separate_cobalt_toolchain = target_os != "android"
 }
 
-assert(!(is_starboard && is_native_target_build),
```

Comment:
Just wondering why this is removed?

Reply:
I misread the conditional in here, I'll add it back